keepa/_version.py

```diff
@@ -1,6 +1,6 @@
 """Version number for keepa
 """
 # major, minor, patch, -extra
-version_info = 0, 20, 1
+version_info = 1, 0, 0
```

Comment:
Probably a good idea to finally get this to `1.0.0` since the API is basically stable at this point.